latest/base.yml

```diff
@@ -103,3 +103,4 @@ helm_chart_repositories:
   cnpg: https://cloudnative-pg.github.io/charts
   codecentric: https://codecentric.github.io/helm-charts
   dnationcloud: https://dnationcloud.github.io/helm-hub
+  rook-release: https://charts.rook.io/release
```

Comment:
Please also add this to base.yml file in the 7.0.0c directory.

Reply:
done